File: 07-fizz-buzz/fizz-buzz.js
// Write code to loop through the array of numbers
// At each iteration, if a number is evenly divisible by 3 print "Fizz"
// If a number is evenly divisible by 5 print "Buzz"
// If a number is evenly divisible by both 3 AND 5, print "Fizz Buzz"
// If a number is not divisible by 3 or 5, print the number

const fizzBuzz=(arry)=>{

    result=[]

    for(var i=0; i<arry.length; i++){
        if(arry[i]%3==0 && arry[i]%5==0){
            result.push('Fizz Buzz')
        }else if(arry[i]%3==0){
            result.push('Fizz')
        }else if(arry[i]%5==0){
            result.push('Buzz')
        }else{
            result.push(arry[i])
        }
    }
    return result
}

console.log(fizzBuzz([13, 14, 15, 16, 17, 18, 19, 20]))

File: 07-fizz-buzz/fizz_buzz.py
def fizzBuzz(numsList):

    result=[]

    for num in numsList:
        if num%3==0 and num%5==0:
            result.append('Fizz Buzz')
        elif num%3==0:
            result.append('Fizz')
        elif num%5==0:
            result.append('Buzz')
        else: 
            result.append(num)
    
    return result

print(fizzBuzz([13, 14, 15, 16, 17, 18, 19, 20]))